gdal/doc/source/drivers/raster/gtiff.rst

```diff
@@ -474,6 +476,16 @@ Creation Options
    tile/strip with lossless WebP compression, so you may want to increase the
    BLOCKYSIZE value for strip layout.
 
+-  **JXL_LOSSLESS=YES/NO**: Set whether JPEG-XL compression should be lossless
+   (YES, default) or lossy (NO)
+
+-  **JXL_EFFORT=[1-9]**: Level of effort for JPEG-XL compression.
```

Comment:
@tbonfort what do you think about `JXL_LEVEL` to be somehow similar to other compression schema options (ZLEVEL, ZSTD_LEVEL)?

Reply:
I agree it would be more consistent with the existing. However the existing is already inconsistent (i.e. a higher WEBP_LEVEL will give you a heavier image, but a higher ZSTD_LEVEL will give you a lighter one). The jxl ecosystem has already committed to using the "effort" keyword for this (with cute names:  1:lightning 2:thunder 3:falcon
 * 4:cheetah 5:hare 6:wombat 7:squirrel 8:kitten 9:tortoise), I'd be inclined to follow that in gdal